data/oci-migrate-conf.yaml

```diff
@@ -0,0 +1,238 @@
+# ----------------------------------------------------------------------------
+# Configuration and status data with respect to the oci-image-migrate.
+# ----------------------------------------------------------------------------
+#
+# dummy format key
+dummy_format_key : '01234567'
```

Comment:
while scanning through the modules the function will also look into the 'template', some_img_type.py, which has as format key 01234567,  this dummy_format_key will be skipped in the eventual output (to avoid confusion) and the module will be ignored.  the name can be changed..